virttest/qemu_devices/qcontainer.py

```diff
@@ -2243,6 +2243,8 @@ def chardev_define_by_params(self, chardev_id, params, file_name=None):
         _backend = 'socket' if 'socket' in backend else backend
         # Generate -chardev device
         chardev_param = Params({'backend': _backend, 'id': chardev_id})
+        if backend == 'pipe':
+            process.system("mkfifo %s" % file_name)
```

Comment:
Prefer handle this after `chardev_param.update({'path': file_name})`.

Reply:
I mean after update 'path', since the following action is related with 'path'
```
if backend in ['unix_socket', 'file', 'pipe', 'serial', 'tty', 'parallel', 'parport']:
    chardev_param.update({'path': file_name})
    if backend == "pipe":
        process.system("mkfifo %s" % file_name)
```